apps/right-to-rent-check/views/tenant-additional-details.html

```diff
@@ -4,6 +4,9 @@
     {{#fields}}
       {{#renderField}}{{/renderField}}
     {{/fields}}
+    <div id="tenant-additional-details-info">
+      {{#t}}pages.tenant-additional-details.info{{/t}}
+    </div>
```

Comment:
This looks like it should just be a `<p>`, which probably then wouldn't need the additional id attribute and styling to give it a bottom margin, since it'll have one already.

Reply:
Good spot, I've amended now